frontend/src/app/main/ui/workspace/libraries.cljs

```diff
@@ -390,8 +397,65 @@
 (mf/defc updates-tab
   {::mf/wrap-props false}
   [{:keys [file-id file-data libraries]}]
-  (let [libraries      (mf/with-memo [file-data libraries]
-                         (filter #(seq (dwl/assets-need-sync % file-data)) (vals libraries)))
+  (let [summary?*  (mf/use-state true)
+        updating?* (mf/use-state false)
+        summary?   (deref summary?*)
+        updating?  (deref updating?*)
+
+        see-all-assets
+        (fn []
+          (reset! summary?* false))
+
+        extract-assets
```

Comment:
This big function, if it only depends on its arguments, should be defined as toplevel function.

Reply:
Done